crates/sui-framework/packages/dwallet-system/tests/tendermint_lc_tests.move

```diff
@@ -116,11 +116,47 @@ module dwallet_system::lc_tests {
+    }
+    
+    #[test]
+    fun tendermint_update_lc_stress_test() {
+        let scenario = test_scenario::begin(SENDER);
+        let ctx = test_scenario::ctx(&mut scenario);
+	let client = set_up_stress_test(ctx);
+        let header = header_stress_test();	
+        // shouldn't return error
```

Comment:
dont we need to add any assertions here?

Reply:
If this test does not pass, it will return an error. Asserts also add in previous tests for testing extract function.